kadi/commands/core.py

```diff
@@ -549,23 +549,44 @@ def fetch_params(self):
         for cmd in self:
             cmd["params"]
 
-    def get_rltt(self):
-        # Find the RLTT, which should be near the start of the table.
+    def get_rltt(self) -> str | None:
+        """Return the first RLTT (Running Load Termination Time) command in table.
+
+        This is a command of type LOAD_EVENT with
+        event_type=RUNNING_LOAD_TERMINATION_TIME.
+
+        Returns
+        -------
+        str or None
+            RLTT as a date string (e.g. '2012:001:23:59:59.999') or None if
+            there is no RLTT in the table.
+        """
         for cmd in self:
             if (
                 cmd["type"] == "LOAD_EVENT"
-                and cmd["params"]["event_type"] == "RUNNING_LOAD_TERMINATION_TIME"
+                and cmd["params"].get("event_type") == "RUNNING_LOAD_TERMINATION_TIME"
             ):
+                print("RLTT", cmd["date"])
```

Comment:
Leftover debug print?

Reply:
Ugh... thx.

Reply:
Testing doesn't catch stray print statements, I'm not sure if there is a good process here. I guess review is the answer.

Reply:
Is there a ruff option to disallow them?

Reply:
Indeed T201! https://docs.astral.sh/ruff/rules/print/
This might be a good one for many of our core packages but it's likely to cause a lot of noise in a more general context.

Reply:
Just tried this in kadi and it would be manageable. So now you got me thinking about another ruff rampage. Must resist. :smile:. Anyway the offending print statement here is removed.